app/console

```diff
@@ -40,5 +40,13 @@ $backend = new BackendInit($kernel);
 $backend->initialize('Console');
 
 $application = new Application($kernel);
-$application->add(new \Common\Command\ImportLocaleCommand());
+$application->add(new \Console\Locale\ImportLocaleCommand());
+if ($kernel->getContainer()->getParameter('site.multilanguage')) {
```

Comment:
@carakas, wouldn't it be better to enable the `\Console\Locale\EnableLocaleCommand` and integrate the "site.multilanguage" check in there, so you can throw an error to let people know they have to change the parameter... ?